partition/roles/monitoring/prometheus/templates/prometheus.yaml.j2

```diff
@@ -33,14 +33,20 @@ alerting:
 
 {% if prometheus_remote_write %}
 remote_write:
-{{ prometheus_remote_write | to_yaml(indent=2) }}
```

Comment:
I think it's a breaking change? Before that it was not a list and the fields you can use are now limited?

/cc @simcod